students/20th/yongminJeon/westagram/js/main.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+const comment = document.querySelector('.section-article__commentInput');
+
+const leaveComment = function () {
+  const commentValue = comment.value;
+  const ulTag = document.querySelector('.section-article__comment');
+  const listTag = document.createElement('li');
+  ulTag.appendChild(listTag);
+  listTag.innerHTML = `<div class='section-article__commentContents'><p><a class='section-article__fanslink' href='#'>yongmin</a> ${commentValue}</p><button class='section-article__commentLinkButton'><i class='far fa-heart'></i></button></div>`;
```

Comment:
```suggestion
  listTag.innerHTML = `
    <div class='section-article__commentContents'>
      <p>
        <a class='section-article__fanslink' href='#'>yongmin</a>${commentValue}
      </p>
      <button class='section-article__commentLinkButton'>
        <i class='far fa-heart'></i>
      </button>
    </div>`;
```
줄바꿈 하는 것이 가독성 측면에서 훨씬 좋아보입니다!